src/datepicker/calendar.js

```diff
@@ -369,8 +377,11 @@ export default class Calendar extends React.Component<
           />
         </CalendarContainer>,
       );
+      monthList.push(<div key={`month-${i}`}>{monthSubComponents}</div>);
```

Comment:
is the wrapping div required? we try to allow all elements to be overridable

Reply:
`monthKey` on L345 is the same as this line's key prop

Reply:
> is the wrapping div required? we try to allow all elements to be overridable

Unfortunately yes, it's needed for styling but also makes sense semantically since it groups each month within it's own block. Thoughts?

> `monthKey` on L345 is the same as this line's key prop

Good call out, will make changes

Reply:
It's probably fine for now. if overriding this div is necessary for someone, we can add in the override later